krux_marathon_api/marathonapi.py

```diff
@@ -107,86 +96,70 @@ def assign_config_data(self, config_file_data, marathon_app_result):
+        """
+        Connects to the Marathon Server using the supplied App ID.
+        If we passed the config file and the App isn't found on the server, we
+        try to create it.
+        If the App name was passed on the command line instead, we will just
+        return with an error if the App wasn't found.
+        Input is marathon connection, config file data, and App id.
+        Returns the Marathon App results.
+        """
+        self.logger.info("Getting app info from the marathon server")
+        if config_file_data:
             try:
-                marathon_app_result = marathon_server.get_app(config_file_data["id"])
+                marathon_app_result = marathon_server.get_app(app_id)
             except Exception as e:
-                self.logger.warn("App doesn't exist %s. Exception is %s" % (config_file_data["id"], e))
+                self.logger.warn("App doesn't exist %s. Exception is %s" % (app_id, e))
                 ### App doesn't exist; initialize it
                 self.create_marathon_app(marathon_server, config_file_data)
                 marathon_app_result = marathon_server.get_app(config_file_data["id"])
-        elif self.args.app:
+        else:
             try:
-                marathon_app_result = marathon_server.get_app(config_file_data)
+                marathon_app_result = marathon_server.get_app(app_id)
             except Exception as e:
-                self.logger.warn("App doesn't exist %s. Exception is %s" % (config_file_data, e))
+                self.logger.warn("App doesn't exist %s. Exception is %s" % (app_id, e))
                 sys.exit(1)
 
-        print(marathon_app_result.id)
         return marathon_app_result
 
     def update_marathon_app(self, marathon_server, config_file_data, marathon_app_result):
+        """
+        Connects to the Marathon Server and sends the App variables. If the
+        values have changed, the App will be updated on the server, otherwise
+        it makes no changes.
+        """
         ### API call to marathon server to update the app if the values have changed
-        self.logger.info("Update marthon server with updated app data")
+        ### The Marathon package and server API does verification of data so no
+        ### reason to recheck here.
+        self.logger.info("Update marathon server with updated app data")
         marathon_server.update_app(config_file_data["id"], marathon_app_result, force=False, minimal=True)
 
     def create_marathon_app(self, marathon_server, config_file_data):
+        """
+        Creates the App name on the Marathon Server with minimal default values.
+        """
         self.logger.info("App wasn't found. Creating new marathon app")
-        print("Creating marathon app.")
         self.logger.info(config_file_data["id"])
 
         ### Initialize app creation and name space
         marathon_server.create_app(config_file_data["id"], MarathonApp(cmd='test', mem=1, cpus=.01))
```

Comment:
`MarathonApp(cmd='test', mem=1, cpus=.01)` as a dummy? If so, this should be clearly documented. Or we can take in `cmd`, `mem`, and `cpus` parameters and give them default values. Or we can take in a config/dict that specifies these values.